java-frontend/src/test/java/org/sonar/java/collections/PCollectionsTest.java

```diff
@@ -19,27 +19,27 @@
  */
 package org.sonar.java.collections;
 
+import org.fest.assertions.Assertions;
 import org.junit.Test;
 
 import java.lang.reflect.Constructor;
 import java.lang.reflect.Modifier;
 
-import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
Why remove this static import ? that generates modification irrelevant to your commit.